.github/workflows/check.yml

```diff
@@ -13,19 +13,19 @@ jobs:
     steps:
       - uses: actions/checkout@v1
 
-      - name: Cache Cargo Registry
-        uses: actions/cache@v1
-        with:
-          path: ~/.cargo/registry
-          key: ${{ runner.os }}-wasm-registry-${{ hashFiles('src/rust/Cargo.lock') }}
-          restore-keys: ${{ runner.os }}-wasm-registry
-
-      - name: Cache Wasm
-        uses: actions/cache@v1
-        with:
-          path: src/rust/target
-          key: ${{ runner.os }}-wasm-build-${{ hashFiles('src/rust/Cargo.lock') }}
-          restore-keys: ${{ runner.os }}-wasm-build
+#      - name: Cache Cargo Registry
+#        uses: actions/cache@v1
+#        with:
+#          path: ~/.cargo/registry
+#          key: ${{ runner.os }}-wasm-registry-${{ hashFiles('src/rust/Cargo.lock') }}
+#          restore-keys: ${{ runner.os }}-wasm-registry
+#
+#      - name: Cache Wasm
+#        uses: actions/cache@v1
+#        with:
+#          path: src/rust/target
+#          key: ${{ runner.os }}-wasm-build-${{ hashFiles('src/rust/Cargo.lock') }}
+#          restore-keys: ${{ runner.os }}-wasm-build
```

Comment:
If this needs to go, it should probably be removed entirely instead of being commented out.

Reply:
I also think that the code should be removed instead of commenting. If we would like to restore it some day, we can just look for the commit.

Reply:
Ok 👌